project_task_stage_state/models/project_task_type.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014 Daniel Reis
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import fields, models
+
+_TASK_STATE = [
+    ("draft", "New"),
+    ("open", "In Progress"),
+    ("pending", "Pending"),
+    ("done", "Done"),
+    ("cancelled", "Cancelled"),
+]
+
+
+class ProjectTaskType(models.Model):
+    """Added state in the Project Task Type."""
+
+    _inherit = "project.task.type"
+
+    state = fields.Selection(_TASK_STATE)
```

Comment:
Could you attend https://github.com/OCA/project/pull/1009#discussion_r1017752736 ?